darwin/dataset/upload_manager.py

```diff
@@ -215,66 +233,46 @@ def upload_file_to_s3(
 
     Returns
     -------
-    dict
-        Key and Id of the image to upload
+    requests.Response
+        s3 response
     """
     key = file["key"]
     image_id = file["id"]
-    response = sign_upload(client, image_id, key, Path(file_path))
+    response = sign_upload(client, image_id, key, file_path)
     signature = response["signature"]
     end_point = response["postEndpoint"]
-
-    s3_response = upload_to_s3(signature, end_point, file_path)
-    # if not str(s3_response.status_code).startswith("2"):
-    #     process_response(s3_response)
-
-    if s3_response.status_code == 400:
-        print(f"Detail: Bad request when uploading to AWS S3 -- file: {file_path}")
-
-    return {"key": key, "id": image_id}
-
-
-def upload_to_s3(signature, end_point, file_path=None):
-    """
-
-    Parameters
-    ----------
-    signature
-    end_point
-    file_path
-
-    Returns
-    -------
-    requests.Response
-    Response of the post request
-    """
-    test = {"file": open(file_path, "rb")}
-    return requests.post("http:" + end_point, data=signature, files=test)
+    return requests.post("http:" + end_point, data=signature, files={"file": file_path.open('rb')})
```

Comment:
Perhaps it would be good practice to actually determine which response we get, and do something in the case of 4** or 5** status codes

Reply:
These gets logged later on in the pipeline